.github/actions/bumpVersion/index.js

```diff
@@ -38,8 +38,15 @@ function updateNativeVersions(version) {
 
     // Update iOS
     updateiOSVersion(version)
-        .then(() => {
-            console.log('Successfully updated iOS!');
+        .then((promiseValues) => {
+            // The first promiseValue will be the CFBundleVersion, so confirm it has 4 parts before setting the env var
```

Comment:
1) I'm not entirely sure we need to double-check the version update here. The unit tests should be good enough, and this code is really pretty simple: 

```javascript
const cfVersion = version.includes('-') ? version.replace('-', '.') : `${version}.0`;
```

2) If you _do_ want to leave this check in place, then I think it would be internal in `updateiOSVersion`. If the `cfVersion` does not have 4 parts, throw an error which would be caught out here. I don't think there's any need to output the `NEW_IOS_VERSION` to the workflow.

Reply:
oops, I'm looking at the compiled file... but yeah still that comment ^

Reply:
My main reason for sending the `NEW_IOS_VERSION` value to the workflow was because of https://github.com/Expensify/Expensify.cash/pull/1693/files#diff-fc40e39603744467ec4d9d7cb6c3bd67756ec09c037d03f3655904fc1c6bcfdaL71. Perhaps I'm making an incorrect assumption here, since `incrementVersion` will return the JS version, don't we need to ensure the version set in ios.yml is the cfVersion set by the logic you posted above?

I also wanted to avoid using `core.setOutput` in another file (currently it's only being set in bumpVersion.js), but if nativeVersionUpdater is only being called in that flow, I'm fine moving it there if it works.